src/Bundle/BundleMetadata.php

```diff
@@ -40,6 +40,11 @@ class BundleMetadata
      */
     protected $name;
 
+    /**
+     * @var string
+     */
+    protected $application;
```

Comment:
please make this private

Reply:
everything else was protected so I just went with it.